microfs.py

```diff
@@ -87,8 +87,15 @@ def flush(serial):
         time.sleep(0.01)
     flush(serial)
     # Go into raw mode with CTRL-A.
-    serial.write(b"\r\x01")
-    flush_to_msg(serial, raw_repl_msg)
+    for i in range(3):
+        # retry to enter raw repl
+        try:
+            serial.write(b"\r\x01")
+            flush_to_msg(serial, raw_repl_msg)
+            break
+        except:
```

Comment:
How about?

```python
except Exception as ex:
    if i == 2:
        raise ex
```

Thus ensuring the exception is raised, even after the three attempts.